front/public/locales/de/operationalStudies/project.json

```
@@ -0,0 +1,82 @@
+{
+  "addImage": "Bild hinzufügen",
+  "budget": "Budget",
+  "createdOn": "Erstellt am ",
+  "createStudy": "Studie erstellen",
+  "deleteItems": "Löschen",
+  "deleteStudies": "Löschen",
+  "description": "Beschreibung",
+  "error": {
+    "unableToDelete": "Löschen nicht möglich.",
+    "unableToDeleteProject": "Projekt löschen nicht möglich.",
+    "unableToDeleteStudy": "Studie löschen nicht möglich.",
+    "unableToPostDocumentTitle": "Bild hochladen nicht möglich",
+    "unableToPostDocument": "Bild laden nicht möglich.",
+    "unableToUpdateProjectTitle": "Projekt aktualisieren nicht möglich",
+    "unableToUpdateProject": "Projekt aktualisieren nicht möglich.",
+    "unableToCreateProjectTitle": "Projekt erstellen nicht möglich",
+    "unableToCreateProject": "Projekt erstellen nicht möglich.",
+    "uploadImageSizeTitle": "Bilddatei zu groß",
+    "uploadImageSize": "Das Bild darf nicht mehr als 2 MB groß sein.",
+    "uploadImageTypeTitle": "Dateiart nicht unterstützt",
+    "uploadImageType": "Nur jpg, jpeg und png werden als Format unterstützt."
```

I didn’t check on the app, but French and English have no final dot

```suggestion
    "unableToDelete": "Löschen nicht möglich",
    "unableToDeleteProject": "Projekt löschen nicht möglich",
    "unableToDeleteStudy": "Studie löschen nicht möglich",
    "unableToPostDocumentTitle": "Bild hochladen nicht möglich",
    "unableToPostDocument": "Bild laden nicht möglich",
    "unableToUpdateProjectTitle": "Projekt aktualisieren nicht möglich",
    "unableToUpdateProject": "Projekt aktualisieren nicht möglich",
    "unableToCreateProjectTitle": "Projekt erstellen nicht möglich",
    "unableToCreateProject": "Projekt erstellen nicht möglich",
    "uploadImageSizeTitle": "Bilddatei zu groß",
    "uploadImageSize": "Das Bild darf nicht mehr als 2 MB groß sein",
    "uploadImageTypeTitle": "Dateiart nicht unterstützt",
    "uploadImageType": "Nur jpg, jpeg und png werden als Format unterstützt"
```